issues/davjs/MissionParms/01-mission-params.md

```diff
@@ -20,22 +20,41 @@ This repo contains the DAV JavaScript SDK. This SDK allows developers to build a
 
 ## How you can help
 
-### The Issue
+In order to foster a community that is welcoming for open source contributions, it is important for us to have good test coverage. And good tests are simple, readable tests.
+
+Here is a good opportunity to simplify one of our tests.
 
+### The Issue
 
+In `src/ride-hailing/MissionParams.test.ts` we defined the following parameters at the beginning of the `describe()` block.
 
+```javascript
+const missionParams = new MissionParams({});
 
+const serializedMissionParams: any = {
+    ttl: undefined,
+    protocol: 'ride_hailing',
+    type: 'mission',
+    price: undefined,
+    vehicleId: undefined,
+};
+```
 
+These parameters will need to accommodate additional unit tests. 
 
+Please update these parameters to be more dynamic by defining them to be mutable similar to `src/vessel-charging/MissionParams.test.ts`.
```

Comment:
`ride-hailing` not `vessel-charging`

Reply:
What helped me understand this issue was the example you pointed me to at `src/vessel-charging/MissionParams.test.ts`. My thought is to provide the same example to a collaborator for additional context of what the task is asking for.

After re-reading this line. I would like to update it to:
> Please update these parameters to be dynamic and mutable. A similar example can be found in `src/vessel-charging/MissionParams.test.ts`.

What do you think?

Reply:
Right. My mistake.
Looks good.